sale_order_mass_action/__openerp__.py

```diff
@@ -0,0 +1,49 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Sale - Sale Order Mass Action module for Odoo
+#    Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Sale Orders Mass Action',
+    'version': '1.0',
+    'author': "GRAP,Odoo Community Association (OCA)",
+    'website': 'http://www.grap.coop',
+    'license': 'AGPL-3',
+    'category': 'Sales',
+    'description': """
+Sale Orders Mass Action
+========================
+
+Facilities to realize mass action on Sale Orders:
+
+* A wizard which allows on multiple Sale Orders at a time to:
+    * confirm Sale Orders;
+
+Possible Improvements:
```

Comment:
Put in Know issues / Roadmap, and follow OCA's template for the rest of the usual sections: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst